config/environments/test.rb

```diff
@@ -14,7 +14,8 @@
   config.action_controller.allow_forgery_protection    = false
 
   config.action_mailer.delivery_method = :test
-  config.action_mailer.default_url_options = { host: "localhost:3000" } # This doesn't work on Cloud9.io
+  # config.action_mailer.default_url_options = { host: "localhost:3000" } # This doesn't work on Cloud9.io
+  config.action_mailer.default_url_options = { host: 'localhost:52662' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.